composer.json

```diff
@@ -15,12 +15,12 @@
         }
     ],
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
```

Comment:
This is 8.1 but the matrix is testing 8.2 and 8.3. We should either bump this to 8.2 also or we should expand the matrix to also test 8.1.